internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -237,6 +238,49 @@ func SetRLimit(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFun
 	}, "setRLimit"
 }
 
+// DropCapabilities drops some capabilities so that they can't be restored by child processes.
+func DropCapabilities(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFunc, string) {
+	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) error {
+		// Disallow raising ambient capabilities (ever).
+		secbits := cap.GetSecbits()
+		secbits |=
+			cap.SecbitNoCapAmbientRaise | cap.SecbitNoCapAmbientRaiseLocked
+
+		if err := secbits.Set(); err != nil {
+			return fmt.Errorf("error setting secbits: %w", err)
+		}
+
+		// Set PR_SET_NO_NEW_PRIVS to limit setuid and similar privilege raising techniques.
+		// See https://www.kernel.org/doc/html/v5.10/userspace-api/no_new_privs.html.
+		const PR_SET_NO_NEW_PRIVS = 38 //nolint:golint,revive,stylecheck
```

Comment:
That's [`unix.PR_SET_NO_NEW_PRIVS`](https://pkg.go.dev/golang.org/x/sys/unix).

Reply:
thanks, not sure how I missed that